.github/workflows/BuildMaster.yml

```
@@ -13,18 +13,18 @@
     name: Build master on ubuntu-latest
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+      - uses: actions/checkout@v1
```

actionlint (1.7.7)

the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)